src/js/videojs-dash.js

```diff
@@ -1,47 +1,45 @@
-'use strict';
-var window_ = require('global/window');
-var videojs = require('video.js');
-var dashjs = require('dashjs');
-
-  var
-    isArray = function(a) {
-      return Object.prototype.toString.call(a) === '[object Array]';
-    },
-    isObject = function (a) {
-      return Object.prototype.toString.call(a) === '[object Object]';
-    },
-    mergeOptions = function(obj1, obj2){
-      var key, val1, val2, res;
-
-      // make a copy of obj1 so we're not overwriting original values.
-      // like prototype.options_ and all sub options objects
-      res = {};
-
-      for (key in obj2){
-        if (obj2.hasOwnProperty(key)) {
-          val1 = obj1[key];
-          val2 = obj2[key];
-
-          // Check if both properties are pure objects and do a deep merge if so
-          if (isObject(val1) && isObject(val2)) {
-            obj1[key] = mergeOptions(val1, val2);
-          } else {
-            obj1[key] = obj2[key];
-          }
+import window from 'global/window';
+import videojs from 'video.js';
+import dashjs from 'dashjs';
```

Comment:
Yeah but isn't it the case that if you `import dashjs from 'dashjs'` here, that it will be bundled with the bundle automatically by browserify? Or am I mistaken?